### PR TITLE
Azure Container Instances: Simple Linux container with public IP

### PR DESCRIPTION
**Pull request by seanmck**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* Initial submission of a simple template for creating a Linux container with a public IP in Azure Container Instances
*
*

### CODE REVIEW

**Comment by seanmck** [comment]
@singhkays Am I missing anything required to merge this?

File: 101-aci-linuxcontainer-public-ip/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {    
+  },
+  "variables": {
+    "name": "acilinuxpublicipcontainergroup",
+    "image": "microsoft/aci-helloworld",   
+    "port": "80",
```
Comment (singhkays) [inline 101-aci-linuxcontainer-public-ip/azuredeploy.json]: Shouldn't these be parameters with these values as default?

Comment (seanmck) [inline 101-aci-linuxcontainer-public-ip/azuredeploy.json]: I was following the best practices doc, which indicates that variables should be preferred over params except for things that must be unique or are secret:

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#best-practices

I can change it to params if you think that's better.

Comment (singhkays) [inline 101-aci-linuxcontainer-public-ip/azuredeploy.json]: That's mostly for unnecessary parameters that folks use such as nic name, lb name. These seem like they would be useful to have for choice when deploying the template and affect the container directly

**Comment by seanmck** [comment]
@singhkays: ACK on changing from vars to params. I've done that, set defaults for all, and included name and image (the most likely to be changed) on the separate params file.

**Pull request merged**